vike-react/renderer/onRenderClient.tsx

```diff
@@ -9,8 +9,10 @@ let root: ReactDOM.Root
 async function onRenderClient(pageContext: PageContextClient) {
   const page = getPageElement(pageContext)
 
+  const ssr = pageContext.config.ssr === false ? false : true
```

Comment:
Interestingly this works, i.e `pageContext.config.ssr` receives here the value that I explicitly set in https://github.com/vikejs/vike-react/blob/main/examples/basic/pages/%2Bconfig.h.ts#L16 , although I set `env` to `server-only` in [renderer/+config.ts](https://github.com/vikejs/vike-react/pull/5/files#diff-543d3390daaa9c91b0973411ea9e9e65ba966a33d1d64f030c166a42d49a3fff). Is this the intended behavior? 

In otherwords, for the new `ssr` flag, there seems to be no difference in behavior between `server-only` and `server-and-client`

Reply:
I was actually surprised about this as well. That's unexpected and I'll have a look at it.

Reply:
I could reproduce. Seems like a bug, let me fix it.

Reply:
Actually I'll release the fix later (because the fix needs to be part of 
an internal refactor which is work-in-progress). Let me know if it turns out to be a blocker. Thanks for the pointer.

Reply:
I didn't try, but I'm fairly confident it's fixed in `0.4.141`.

Reply:
Yes, I confirm it's fixed in `0.4.141`. I just validated locally, playing with `config-only`, `server-only` and `server-and-client`. Thanks!